dev-python/grpcio/grpcio-1.66.2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="High-performance RPC framework (python libraries)"
+HOMEPAGE="https://grpc.io"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+RDEPEND="
+	dev-libs/openssl[-bindist(-)]
+	dev-libs/re2
+	dev-python/protobuf-python[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	net-dns/c-ares
+	sys-libs/zlib
+"
+
+BDEPEND="${RDEPEND}
```

Comment:
Are you convinced that all of these are `BDEPEND`? Why would they?

Also Cython suggests you're building a C extension, in which case `DISTUTILS_EXT` would be necessary. And I'm pretty sure the eclass will output a warning about that.

Reply:
Fixed

Reply:
Is protobuf-python a BDEPEND?